lib/voyager/clients/twitter_client.rb

```diff
@@ -183,6 +183,18 @@ def recent_tweets(options={})
       get("/statuses/user_timeline.json", options)
     end
 
+    # ============================================================================
+    # Metrics Methods - These act on Tweets
+    # ============================================================================

```

Comment:
Since you are in here anyway I would just throw in your recent tweets method as well, even though we may not use it:

```
def recent_tweets(options={})
   get('/tweets/search/recent', options)
 end
```